backend/server.js

```diff
@@ -5,18 +5,16 @@ import dotenv from 'dotenv'
 import ratingRouter from './routes/ratings_route.js'
 import placesRouter from './routes/places_route.js'
 import userRouter from './routes/users_route.js'
-
+import businessRouter from './routes/business_route.js';
```

Comment:
I fixed this in server.js
There were double imports. Now it is good after I updated